code/modules/mob/living/carbon/human/human.dm

```diff
@@ -894,9 +894,14 @@
 	if(istype(location, /turf))
 		location.add_vomit_floor(src, 1)
 
+	if(reagents.reagent_list.len)
+		var/purge_percent = 0.7 //30%
+		for(var/datum/reagent/ingested_chem in reagents.reagent_list)
```

Comment:
as anything

Reply:
for(var/datum/reagent/ingested_chem as anything in reagents.reagent_list)